.github/workflows/action.yml

```diff
@@ -137,6 +137,22 @@ jobs:
         name: python
         path: dist
 
+  python_release:
+    needs: [build-python]
+    runs-on: ubuntu-latest
+    if: "startsWith(github.ref, 'refs/tags/')"
+    steps:
+      - uses: actions/download-artifact@v3
+        with:
+          name: python
+      - name: Publish to PyPI
```

Comment:
why not ? https://github.com/pypa/gh-action-pypi-publish
Looks safer since it uses thrusted publishers https://blog.pypi.org/posts/2023-04-20-introducing-trusted-publishers/

Reply:
ping @RaulTrombin